src/main/java/Deadline.java

```diff
@@ -0,0 +1,52 @@
+import java.text.ParseException;
+import java.util.Date;
+import java.text.SimpleDateFormat;
+
+public class Deadline extends Task {
+    private Date date;
+    private String dateToStorage;
```

Comment:
This variable name might require further explanation, consider changing this to a more obvious one